channels/in.m3u

```diff
@@ -400,68 +410,38 @@ http://158.69.124.9:1935/sardaritv/sardaritv/playlist.m3u8
 https://satsangtv.sanskargroup.in/satsangtvlive.m3u8
 #EXTINF:-1 tvg-id="ServeurTV.in" tvg-name="Serveur TV" tvg-country="IN" tvg-language="English" tvg-logo="" group-title="",Serveur TV
 https://a.jsrdn.com/broadcast/060753d37e/+0000/high/c.m3u8
-#EXTINF:-1 tvg-id="ShemarooBollywoodClassic.in" tvg-name="Shemaroo Bollywood Classic" tvg-country="IN" tvg-language="Hindi" tvg-logo="https://i.imgur.com/cMJVuIr.jpg" group-title="Movies",Shemaroo Bollywood Classic
-https://livechannel.shemaroome.com/linearplayout/02-bollywood-classic/chunklist_1920x1080_cf.m3u8
```

Comment:
Source works. Was it a part of some cleanup?

Reply:
Yes. It's the same chunk of program over and over again. If you look at the root URL you'll see files that weren't updated since 2019/2020.